tests/test_api.py

```diff
@@ -299,9 +304,8 @@ def test_check_ref_detailed(scm: Git, artifact: str):
     )
 
 
-def test_check_ref_multiple_showcase(scm: Git, showcase: Tuple[str, str]):
-    first_commit, second_commit = showcase
-
+@pytest.mark.usefixtures("showcase")
+def test_check_ref_multiple_showcase(scm: Git):
```

Comment:
this is unrelated to the bug, but resolves pylint error from dropping `disable=unused-variable` in this file